apps/docs-snippets/package.json

```diff
@@ -4,8 +4,9 @@
   "description": "",
   "private": true,
   "scripts": {
-    "pretest": "pnpm build:forc",
-    "build:forc": "pnpm fuels build"
+    "pretest": "run-s build:forc type:check",
+    "build:forc": "pnpm fuels build",
+    "type:check": "tsc --noEmit"
```

Comment:
Is it still necessary to have these `type:check` commands given https://github.com/FuelLabs/fuels-ts/pull/3046#discussion_r1741591923?

Reply:
I kept this one and [fuel-gauge](https://github.com/FuelLabs/fuels-ts/blob/master/packages/fuel-gauge/package.json), as we may have `.ts` files within these packages that wouldn't get picked up because we don't build.